src/core/fft_helpers.cpp

```diff
@@ -134,7 +134,7 @@ int compressbands(float *absspec_buffer, float *compressedband, int num_old, int
 
 	ratio=(float)usefromold/(float)num_new;
 
-	// foreach new subband
+	// for each new subband
```

Comment:
I wondered if anyone would spot that one :grin:. I really wanted `grep -r foreach` to be clean :yum: